torch/generic/Tensor.c

```diff
@@ -240,6 +240,22 @@ static int torch_Tensor_(clone)(lua_State *L)
   return 1;
 }
 
+static int torch_Tensor_(cloneOn)(lua_State *L)
```

Comment:
The one reason I like the given behavior is that there might be clone()
calls buried inside modules, that I still want to work in auto-mode.

On Fri, May 1, 2015 at 11:39 AM, Dominik Grewe <notifications@github.com>
wrote:

> In torch/generic/Tensor.c
> <https://github.com/torch/cutorch/pull/144#discussion_r29508995>:
>
> > @@ -240,6 +240,22 @@ static int torch_Tensor_(clone)(lua_State *L)
> >    return 1;
> >  }
> >
> > +static int torch_Tensor_(cloneOn)(lua_State *L)
>
> Do we still need the logic in torch_Tensor_(clone) (above) for
> auto-device mode? Maybe we should just assert that we're in manual mode
> (same for cuda), which would be more in line with the documentation.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/torch/cutorch/pull/144/files#r29508995>.
>